server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -419,9 +419,15 @@ def _apply_and_create_v3io_source(
             != mm_constants.MonitoringFunctionNames.APPLICATION_CONTROLLER
             else mlrun.mlconf.get_v3io_access_key()
         )
+        explicit_ack_mode = (
+            "explicitOnly"
+            if function_name
+            != mm_constants.MonitoringFunctionNames.APPLICATION_CONTROLLER
+            else "disable"
+        )
```

Comment:
```suggestion
```
It's unnecessary to set it to disable, and in fact we don't do it when `mlrun.mlconf.is_explicit_ack_enabled()` returns false. Would be enough to just change the condition:
```python
        if (
            mlrun.mlconf.is_explicit_ack_enabled()
            and function_name
            != mm_constants.MonitoringFunctionNames.APPLICATION_CONTROLLER
        ):
```
Disabling in two different ways is misleading.